core/src/main/scala/org/apache/spark/storage/PushBasedFetchHelper.scala

```diff
@@ -197,9 +197,12 @@ private class PushBasedFetchHelper(
           localShuffleMergerBlockMgrId)
       }
     } else {
-      logDebug(s"Asynchronous fetch the push-merged-local blocks without cached merged dirs")
-      hostLocalDirManager.getHostLocalDirs(localShuffleMergerBlockMgrId.host,
-        localShuffleMergerBlockMgrId.port, Array(SHUFFLE_MERGER_IDENTIFIER)) {
+      // Push-based shuffle is only enabled when the ESS is enabled. If ESS is not enabled, then
```

Comment:
Shall we use `external shuffle service` instead of `ESS`? Someone uses ESS as external shuffle storage.

Reply:
For now, this file has another instance of ESS. It would be great if you can switch it together, @otterc .
```
$ git grep ' ESS '
core/src/main/scala/org/apache/spark/storage/PushBasedFetchHelper.scala: * shuffle blocks that belong to the common reduce partition and were merged by the ESS to that
```

Reply:
Switched all the references to ESS which were introduced with  #32140